docker-compose.yml

```diff
@@ -250,8 +249,40 @@ services:
             - ./docker/scripts:/scripts
         command: /scripts/run-mongodb-encrypted.sh
 
+    standalone-backup:
+        container_name: "standalone-backup"
+        build:
+            dockerfile: ./docker/mongodb-auth.dockerfile
+            args:
+                TEST_MONGODB_IMAGE: mongo:4.4
+        ports:
+            - 27037:27017
+        expose:
+            -   "27017"
+        environment:
+            - MONGO_INITDB_ROOT_PASSWORD=pbm
+            - MONGO_INITDB_ROOT_USERNAME=pbm
+        command: [ "--replSet", "standaloneBackup", "--bind_ip_all", "--port", "27017", "--keyFile", "/opt/keyfile" ]
+        networks:
+        - standalone-backup
+
+    percona-backup-mongodb:
+        image: percona/percona-backup-mongodb:2.5.0-multi
+        container_name: "percona-backup-mongodb"
+        # We attempt to restart otherwise, PBM will exit since replication is not set up on mongo instance yet
+        restart: on-failure:5
+        environment:
+            - PBM_MONGODB_URI=mongodb://pbm:pbm@standalone-backup:27017
+        volumes:
+            - ./docker/pbm/config:/etc/config
+            - ./docker/pbm/backups:/opt/backups
```

Comment:
Yeah this seemed more straightforward (since pbm needs write access to the folder). With volumes, we might need a custom image (or another solution) to give pbm the same write access.